frontend/package-lock.json

```diff
@@ -1191,6 +1191,7 @@
       "version": "2.6.1",
       "resolved": "https://registry.npmjs.org/async/-/async-2.6.1.tgz",
       "integrity": "sha512-fNEiL2+AZt6AlAw/29Cr0UDe4sRAHCpEHh54WMz+Bb7QfNcFw4h3loofyJpLeQs4Yx7yuqu/2dLgM5hKOs6HlQ==",
+      "optional": true,
```

Comment:
I ran NPM CI so I'm not sure where this came from.

Reply:
Very strange. Maybe you have a different version of `npm`? This shouldn't be a big deal, so I'll mark this as resolved. I just wanted to highlight this in case you were using the wrong command